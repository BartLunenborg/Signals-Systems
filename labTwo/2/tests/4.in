2: [-10,47]
2: [-30,12]
100: [39,24,-12,-30,-4,-48,-43,28,9,-32,-42,-23,2,13,-18,21,-47,36,-17,-44,8,5,29,-14,-10,16,-40,30,13,-20,-8,-46,6,-18,-26,-48,37,-19,-18,-4,-48,42,-24,4,6,8,-24,-39,-6,-39,19,-48,17,-50,41,-43,17,3,-11,-20,-15,-17,-15,42,15,9,44,2,-7,29,1,-5,21,-21,49,29,37,27,40,33,39,9,-12,6,10,-21,-35,29,-16,-44,-41,19,-11,46,11,6,6,-42,11,1]
