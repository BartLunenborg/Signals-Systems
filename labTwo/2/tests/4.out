102: [11700,-52470,-18324,22896,37932,-25200,58284,47118,-64392,-7578,41436,39312,12102,-12132,-24162,41172,-56382,94554,-86688,33114,60132,-35556,5562,-45750,34776,12204,-42120,79224,-64560,-8970,35532,-12840,67668,-40524,23124,15228,69876,-89382,44538,15624,-18432,83784,-98532,61608,-17856,-4524,-16056,29532,44334,-24516,61986,-65466,89256,-68082,98388,-103830,94014,-49362,1698,12522,19896,6570,13050,25962,-68220,3438,7890,-61644,19656,20538,-48018,13326,14514,-41250,58674,-78114,-5634,-32154,-8442,-36072,-16230,-38358,4626,25236,-12948,-18216,27270,50406,-68910,27636,45996,43614,-55494,41346,-73284,10914,-1176,-18396,70944,-40218,4674,564]
